ci-operator/step-registry/openstack/test/dpdk/openstack-test-dpdk-commands.sh

```diff
@@ -80,21 +80,27 @@ EOF
 )
 echo "Created \"$CNF_NAMESPACE\" Namespace"
 
-if ! openstack network show "${OPENSTACK_DPDK_NETWORK}" >/dev/null 2>&1; then
-    echo "Network ${OPENSTACK_DPDK_NETWORK} doesn't exist"
-    exit 1
-fi
-
-cat <<EOF > "${SHARED_DIR}/additionalnetwork.yaml"
+if oc get SriovNetworkNodePolicy/dpdk1 -n openshift-sriov-network-operator >/dev/null 2>&1; then
+    echo "DPDK network is managed by SriovNetworkNodePolicy/dpdk1, CNI won't be changed"
```

Comment:
So a Pod created on that host will already have the new interface configured correctly?

I think we need update on the README https://github.com/openshift/sriov-network-operator/blob/184af61a5406c2c6298eb88c10303d90d5266c76/README.md#sriovnetworknodepolicy.
Let me know what you think and I will try to do.

Reply:
Yes. However, why the CNI settings are not needed anymore when the SriovNetworkNodePolicy is available? I'm just wondering at which step the Pod will be configured with the new nic